doc/whats_new/v0.22.rst

```diff
@@ -57,6 +57,16 @@ Changelog
   ``decision_function_shape='ovr'``, and the number of target classes > 2.
   :pr:`12557` by `Adrin Jalali`_.
 
+
+:mod:`sklearn.cluster`
+..................
+
+- |Fix| :class:`cluster.SpectralClustering` now accepts a ``n_components`` 
```

Comment:
I'd call this an enhancement. Fix indicates that someone's code was previously not functioning as documented, ordinarily. Rather, you provide a way to access existing capabilities

Reply:
Thanks for letting me know. Just made the change